daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/dpath/NodeInfo.scala

```diff
@@ -387,7 +387,7 @@ object NodeInfo extends Enum {
   }
 
   /**
-   * For things like fn:exists, fn:empty, dfdl:contenLength
+   * For things like fn:exists, fn:empty
```

Comment:
Not sure this comment should be changed. I think dfdl:contentLength and dfdl:valueLength should still be using NodeInfo.Exists as their required type for the first argument.

Reply:
Thanks for catching that one. Updated.